security/citadel/templates/deployment.yaml

```diff
@@ -35,7 +35,11 @@ spec:
           args:
             - --append-dns-names=true
             - --grpc-port=8060
+          {{- if and (.Values.global.istioVersion) (semverCompare "<1.2" .Values.global.istioVersion) }}
```

Comment:
Can we revert the change in citadel instead ( or make sure they maintain proper backward compat between releases ?)